src/Controller/ListController.php

```diff
@@ -96,4 +97,18 @@ public function getMembersAction(Request $request, SubscriberList $list): View
 
         return View::create()->setData($list->getSubscribers());
     }
+
+    /**
+     * Gets the total number of subscribers of a list.
+     * @param Request $request
+     * @param SubscriberList $list
+     *
+     * @return View
+     */
+    public function getNumberAction(Request $request, SubscriberList $list): View
```

Comment:
I'd prefer the name `getCount` as "number" could also mean "just some number". What do you think?

Reply:
@oliverklee I agree, getCount sounds better. Thank you for your feedback.